include/deal.II/grid/grid_in.h

```diff
@@ -456,6 +455,44 @@ class GridIn
   void read_tecplot (std::istream &in);
 
   /**
+   * Read in a file supported by Assimp, and generate a Triangulation
+   * out of it.  If you specify a @p mesh_index, only the mesh with
+   * the given index will be extracted, otherwise all meshes which are
+   * present in the file will be used to generate the Triangulation.
+   *
+   * This function can only be used to read two-dimensional meshes (possibly
```

Comment:
pity

Reply:
I don't know if `.obj` supports volume information... so far I could only make it work with surface one. I know it's a pity, but there is a lot of BEM and shell models that would benefit from this anyway. :)